llama_stack/distribution/server/auth_providers.py

```diff
@@ -224,6 +240,12 @@ def validate_claims_mapping(cls, v):
                 raise ValueError(f"claims_mapping value is not a valid attribute: {value}")
         return v
 
+    @model_validator(mode="after")
+    def validate_mode(self) -> Self:
+        if not self.jwks and not self.introspection:
```

Comment:
you should also check that _both_ are not specified

Reply:
Agreed, fixed.